Src/Support/IntegrationTests/ServiceAccountTests.cs

```diff
@@ -74,19 +74,51 @@ public async Task JwtAccessToken_Http()
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         }
 
-        [Fact]
-        public void JwtAccessToken_RestClient()
+        [Theory]
+        [InlineData(true, null)]
+        [InlineData(false, null)]
+        [InlineData(true, new[] { "https://www.googleapis.com/auth/bigtable.admin" })]
+        public void JwtAccessToken_RestClient(bool useJwtAccessWithScopes, string[] scopes)
         {
             // See: https://developers.google.com/identity/protocols/oauth2/service-account#jwt-auth
 
+            GoogleCredential credential = useJwtAccessWithScopes ?
+                Helper.GetServiceCredentialWithJwtFlag().CreateScoped(scopes) 
```

Comment:
Formatting nit, just in terms of how we normally format a multi-line conditional operator.

```suggestion
            GoogleCredential credential = useJwtAccessWithScopes
                ? Helper.GetServiceCredentialWithJwtFlag().CreateScoped(scopes) 
```

Reply:
thanks, fixed!